src/main/java/com/example/bookstore/dto/order/OrderDto.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookstore.dto.order;
+
+import com.example.bookstore.dto.orderitem.OrderItemDto;
+import com.example.bookstore.model.Order.Status;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+public class OrderDto {
+    private Long id;
+    private Long userId;
+    @EqualsAndHashCode.Exclude
```

Comment:
I think we do not need to exclude any fields in DTO classes, it can be a problem in Entity classes though